src/edu/csus/ecs/pc2/core/report/EventFeedJSONReport.java

```diff
@@ -49,7 +50,9 @@ private void writeContestTime(PrintWriter printWriter) {
     @Override
     public void writeReport(PrintWriter printWriter) throws IllegalContestState {
 
-        EventFeedJSON efEventFeedJSON = new EventFeedJSON(contest);
+        // TODO: Should specify which API version the report is for.  In fact, this class should
+        // be duplicated for each API version.  ATM we use 2023-06
```

Comment:
Just a note to point out that this is a significant "TODO"... we need to figure out how to make sure that we remember to "fix" this soon -- otherwise we're going to find ourselves upgrading to a new API version and producing old (invalid) JSON reports.

Reply:
I am very aware of this.